stdlib/public/Reflection/TypeRefBuilder.cpp

```diff
@@ -193,36 +193,85 @@ const TypeRef *TypeRefBuilder::lookupSuperclass(const TypeRef *TR) {
   return Unsubstituted->subst(*this, *SubstMap);
 }
 
+static llvm::Optional<StringRef> FindOutermostModuleName(NodePointer Node) {
+  // Breadth first search until we find the module name so we find the outermost
+  // one.
+  llvm::SmallVector<NodePointer, 8> queue;
+  queue.push_back(Node);
+  // Instead of removing items from the front of the queue we just iterate over
```

Comment:
Looks like you're doing both. Is that intentional?

Reply:
No, that's a mistake, will fix